services/galley/src/Galley/Env.hs

```diff
@@ -104,3 +108,15 @@ currentFanoutLimit o = do
   let optFanoutLimit = fromIntegral . fromRange $ fromMaybe defFanoutLimit (o ^. (optSettings . setMaxFanoutSize))
   let maxTeamSize = fromIntegral (o ^. (optSettings . setMaxTeamSize))
   unsafeRange (min maxTeamSize optFanoutLimit)
+
+mkRabbitMqChannel :: Logger -> Opts -> IO (Maybe (MVar Q.Channel))
+mkRabbitMqChannel l (view optRabbitmq -> Just RabbitMqOpts {..}) = do
```

Comment:
should this go to wire-api?  that would also pull a few other changes there, and clean up the arguments (`Opts` is not available in wire-api, but `RabbitMqOpts` should be).

Reply:
I think `wire-api` is the wrong place to put it as this has nothing to do with the API. I was thinking of shoving it in the `exteded` package, but forgot about it. Let's do it in the next PR?